### DISCUSSION TITLE
Contributing to Github.com

### DISCUSSION THREAD

**Discussion by lmckeen** [community/community#22150] [New to GitHub]
<p>Does anyone know if the code/design for <a href="http://github.com" rel="nofollow noopener">github.com</a> is open-sourced? I would like to contribute :slight_smile:</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>There are a number of components of GitHub that are open-sourced, yes. These include <a href="https://github.com/github/linguist">linguist</a>, the library that we use to detect the language of files in repositories, and <a href="https://github.com/primer">primer</a>, the design framework that the GitHub website uses (though not the entirety of the design itself).</p>
